libraries/AP_Motors/AP_MotorsMulticopter.h

```diff
@@ -126,11 +127,17 @@ class AP_MotorsMulticopter : public AP_Motors {
     // return gain scheduling gain based on voltage and air density
     float               get_compensation_gain() const;
 
-    // convert thrust (0~1) range back to pwm range
-    int16_t             calc_thrust_to_pwm(float thrust_in) const;
+    // convert actuator output (0~1) range to pwm range
+    int16_t             output_to_pwm(float _actuator_output);
+    
+    // converts desired thrust to linearized actuator output in a range of 0~1
+    float               thrust_to_actuator(float _actuator_output);
```

Comment:
Looks like the argument (i.e. _actuator_output) is mismatched between the .h and .cpp.  Also normally we reserve the underscore at the front for internal variables so let's remove the underscore if possible.